### DISCUSSION TITLE
Split string to make elements accessible

### DISCUSSION THREAD

**Discussion by horsto** [bonsai-rx/bonsai#1147] [Q&A] [answered]
I am creating a string in a python transform that I am sending over serial port  to a microcontroller. It comes in the form `X_Y`, where X and Y are two integers. At the same time I want to write the two integer values into a CSV, and therefore want to split the string into the two integers `X` and `Y` and write them into two columns in the CSV. 
Is there a way to do that elegantly - for example with `parse` ?

**Comment by bruno-f-cruz** [comment, marked as answer]
You can use the `Parse` operator with the pattern `%f_%f`, outputting a tuple with two floats.

**Reply by horsto** [reply]
thanks!
